package.json

```diff
@@ -3,12 +3,14 @@
   "version": "0.1.0",
```

Comment:
- [ ] It would be better if we maintain a professional README of our project. Here is a link of the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) that we can modify to meet the requirements